data-es6.js

```diff
@@ -6329,6 +6477,7 @@ exports.tests = [
           note_id: "ie-clampedarray",
           note_html: "A buggy implementation of <code>Uint8ClampedArray</code> on initial IE11 release has been fixed with the patch on <a href='https://support.microsoft.com/en-us/help/2929437/description-of-the-security-update-for-internet-explorer-11-on-windows'>KB2929437</a>"
         },
+        ie11: false,
```

Comment:
1 line up there is ie 11 already with flagged support. Did they broke it again?

Reply:
Ooops.  Fixed.